Mac/BuildScript/build-installer.py

```diff
@@ -61,7 +61,7 @@
 try:
     from plistlib import writePlist
 except ImportError:
-    # We're run using python2.3
+    # Run from python2.3
```

Comment:
Thanks for your interest but the comment looks to me to be as grammatically correct as the suggested replacement and, in any case, not worth changing by itself.  We generally do not make trivial changes like this by themselves so as not to unnecessarily clutter the code base and complicate maintenance. Sorry!